document/endnotes_test.go

```diff
@@ -0,0 +1,285 @@
+package document_test
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/unidoc/unioffice/document"
+)
+
+func TestEndnotesLoad(t *testing.T) {
+	doc := loadTestData(t)
+	if doc == nil {
+		t.Fatal("could not read test document")
+	}
+	saveTestData(t, doc)
+}
+
+func TestHasEndnotes(t *testing.T) {
+	doc := loadTestData(t)
+	if doc == nil {
+		t.Fatal("could not read test document")
+	}
+	if doc.HasEndnotes() == false {
+		t.Error("document should contain endnotes")
+	}
+	doc = document.New()
+	if doc.HasEndnotes() == true {
```

Comment:
Also here `doc.HasEndnotes()` can be used instead of `doc.HasEndnotes() == true`